pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -259,3 +259,22 @@ func getNormalizedNameForSelector(sel *antreatypes.GroupSelector) string {
 	}
 	return ""
 }
+
+func (n *NetworkPolicyController) syncInternalGroup(key string) error {
+	defer n.triggerANPUpdates(key)
+	defer n.triggerCNPUpdates(key)
+	defer n.triggerParentGroupSync(key)
+	// Retrieve the internal Group corresponding to this key.
+	grpObj, found, _ := n.internalGroupStore.Get(key)
+	if !found {
+		klog.V(2).Infof("Internal group %s not found.", key)
+		n.groupingInterface.DeleteGroup(clusterGroupType, key)
```

Comment:
We should rename clusterGroupType given it's used by namespaced group too.

Reply:
Renamed to "internalGroupType" to keep consistent with "internalGroup" elsewhere.